utils/Cargo.toml

```diff
@@ -38,4 +38,10 @@ server = [
 
 [dev-dependencies]
 serde_json = "1.0"
-tokio = { version = "1", features = ["macros", "rt"] }
\ No newline at end of file
+env_logger = "0.9"
+tokio = { version = "1", features = ["macros", "rt"] }
+# TODO: (dp) seems like the macro doesn't work with the component crates, but users must import `jsonrpsee`?
```

Comment:
I don't get this? Can you elaborate?

Reply:
I tried writing a test using the proc macro and only the server, `#[rpc(server)]` and wanted to pull in only the minimum dependencies; I thought all I needed was `jsonrpsee-proc-macros` but that doesn't compile and so I tried a few different combinations but in the end I needed the full` jsonrpsee` façade crate. So the question was: is this a regression?

Reply:
No, we assume that the facade crate is used there I think or maybe my recent refactoring broke that.

Should we still support both?! I still think that this test should be moved to `integration test crate` (not really a unit test IMO)

Reply:
> (not really a unit test IMO)

Well. I think I disagree. The test is unit-testing `call_with`. You are correct that it touches on a bunch of other stuff but that's because the setup is kind of long.